test/Microsoft.ML.Tests/DatabaseLoaderTests.cs

```diff
@@ -233,6 +233,14 @@ public void TestLoadDatetimeColumnWithNullValue()
                 connection.Open();
                 using (var command = new SQLiteCommand(connection))
                 {
+                    // Make sure the table doesn't exist.
+                    command.CommandText = """
+                        BEGIN;
+                        DROP TABLE IF EXISTS Datetime;
+                        COMMIT;
+                        """;
+                    command.ExecuteNonQuery();
+
```

Comment:
Is this a side effect of spitting the test? or this is just fix known issue?

Reply:
It fixes a known issue that I hit while doing this which is why its here. IT doesn't affect CI at all, but it does affect local machines.